ee/packages/omnichannel-services/src/QueueWorker.ts

```diff
@@ -117,6 +120,10 @@ export class QueueWorker extends ServiceClass implements IQueueWorkerService {
 	// This is a "generic" job that allows you to call any service
 	async queueWork<T extends Record<string, unknown>>(queue: Actions, to: string, data: T): Promise<void> {
 		this.logger.info(`Queueing work for ${to}`);
+		if (!this.queueStarted) {
+			this.registerWorkers();
```

Comment:
although `registerWorkers` is `async` it doesn't have any `await` in it, but just in case, should we `await` here?

Reply:
No. It was async cause at the beginning, I created the indexes here. Then removed the async and made it return a promise.

I saw it today returning a promise and made it async...

Cycle of life